### PR TITLE
remove es aggregation,improve es search performance

### PR DESCRIPTION
**Pull request by zwukong**
i have two account,sorry for the bother,please @ JarVzhao ~

### CODE REVIEW

**Comment by zwukong** [comment]
i have tested codes locally, if you have any questions,please @ JarVzhao ~

File: zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/ElasticsearchSpanStore.java [hunk 1]
```diff
@@ -63,6 +60,8 @@
     long beginMillis = Math.max(endMillis - request.lookback(), EARLIEST_MS);
 
     SearchRequest.Filters filters = new SearchRequest.Filters();
+    //only search for SERVER span,which is simpler and faster than aggregation
+    filters.addTerm("kind", "SERVER");
```
Comment (codefromthecrypt) [inline zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/ElasticsearchSpanStore.java:64]: we cannot assume this.. there are spans that are messaging in nature (PRODUCER/CONSUMER). also, RPC spans themselves are optional..

**Review by codefromthecrypt — COMMENTED** [review]
Thanks for the spike.. I'm quite surprised this passed for you... maybe you don't have docker installed?

Tests like https://github.com/openzipkin/zipkin/blob/master/zipkin-storage/elasticsearch/src/test/java/zipkin2/elasticsearch/integration/ITElasticsearchStorageV6.java will need to pass. If you can't run docker, you it should look for localhost elasticsearch...

**Comment by codefromthecrypt** [comment]
right, so I'm sorry the tests will "ignore" if elasticsearch isn't running and docker is not installed. You can see the failure here https://travis-ci.org/openzipkin/zipkin/builds/407517059

First thing is to run the tests then you will know what works or not with this approach. Thanks again for giving a try.

**Comment by zwukong** [comment]
sorry,i forgot to run the unit tests. i use a local es,spans are from sleuth, the ui query is ok.

File: zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/ElasticsearchSpanStore.java (hunk 1 above)
Comment (zwukong) [reply zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/ElasticsearchSpanStore.java:64]: i see,in our company,we use different queries ,for example in messaging circumstance,the kind may be messaging.UI splits them,not combine them all together

Comment (zwukong) [reply zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/ElasticsearchSpanStore.java:64]: aggregation is very slow in es especially in large indices

**Comment by codefromthecrypt** [comment]
are you planning to continue this?

**Comment by codefromthecrypt** [comment]
closing until effort resumes

**Pull request closed without merging**